docs/dev/plugins/cloudFoundry/Basics.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Foundry Plugin
+
+## Basic information of Cloud Foundry (CF)
+CF is a open-source Platform as a Service model (PaaS) which allows everyone to easily deploy an existing application on an infrastructure e.g. AWS, vSphere or OpenStack.   
+CF installs everything what the application needs to compile and run like OS, buildpack with all languages, libraries, and services that the app uses.
```

Comment:
Change `...everything what the application needs...` to `...everything that the application needs...`